### ISSUE TITLE
PDBs not shipped on Windows

### ISSUE THREAD

**Issue by Be-ing** [tenacityteam/tenacity-legacy#595]
### Guidelines

- [X] I have read the guidelines.

### Version/Commit hash

any

### Describe the bug.

Windows builds do not ship PDBs. It is very doubtful we will ever get backtraces from Windows users without this.

### Expected behavior

PDBs for the application are shipped with the builds. PDBs for dependencies are not required.

### OS

Windows

### Additional context

https://github.com/tenacityteam/tenacity/pull/590#issuecomment-915184512

### This issue is not a duplicate

- [X] I have confirmed this issue isn't a duplicate.

**Comment by n0toose** [comment]
Compiling this on a fresh machine using Visual Studio 2022 seems to contain the PDF file. As long as it's the Debug build.

![2021-09-19_12-57](https://user-images.githubusercontent.com/30193966/133925025-da4f5967-58bf-40f4-8344-4317109c578b.png)

**Comment by AnotherFoxGuy** [comment]
@panos The issue is that the PDB isn't included in the windows installer

**Comment by n0toose** [comment]
Yeah, I know, I'm just sort of trying to pinpoint the part that is responsible for including it. Well, for starters, it does exist.

**Comment by AnotherFoxGuy** [comment]
The reason is that you normally exclude PDB's from the installer, because without the sourcecode installed the PDB's are not useful
